x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/rules_table_utility_bar.tsx

```diff
@@ -179,6 +181,16 @@ export const RulesTableUtilityBar = React.memo<RulesTableUtilityBarProps>(
               >
                 {i18n.REFRESH_RULE_POPOVER_LABEL}
               </UtilityBarAction>
+              {rulesTableContext.state.hasSavedState && (
+                <UtilityBarAction
+                  dataTestSubj="clearTableFilters"
+                  iconSide="left"
+                  iconType="cross"
+                  onClick={clearTableState}
+                >
+                  {i18n.CLEAR_RULES_TABLE_FILTERS}
+                </UtilityBarAction>
```

Comment:
It looks a bit counter-intuitive to me, why we show the "clear" button when the state of the table has been saved. I'd expect a check like "if current filters, sorting, or pagination are not the default ones, we show the button". That would decouple (at least partially) the clear button from the persistence feature. One way to think about that: what if we didn't have the persistence feature, but wanted to implement the clear button? Would we be able to do so? I think so.